stable/locust/Chart.yaml

```diff
@@ -1,6 +1,7 @@
+apiVersion: v1
 name: locust
 description: A modern load testing framework
-version: 0.4.0
+version: 1.0.0
```

Comment:
Shouldn't this be `0.4.1`?

Reply:
we agreed some time ago that all charts in the stable folder should be at least 1.0.0 @mattfarina